core/src/main/scala/org/apache/spark/h2o/backends/SparklingBackend.scala

```diff
@@ -34,4 +34,6 @@ trait SparklingBackend {
   def stop(stopSparkContext: Boolean)
 
   def backendUIInfo: Seq[(String, String)]
+
+  def backendFinalisedString: String
```

Comment:
Minor: weird name for variable - `closingString`? Or preamble antonym - `epilog`?

Reply:
Thanks!  I really like `epilog`